apps/emqx_auth_pgsql/src/emqx_auth_pgsql_cli.erl

```diff
@@ -56,6 +56,7 @@ pgvar(Sql, Params) ->
 %% PostgreSQL Connect/Query
 %%--------------------------------------------------------------------
 
+-dialyzer(no_match).
```

Comment:
this will suppress all warning in this module. please use  `-dialyzer([{nowarn_function, [connect/1]}]).` instead.

Reply:
also add a comment to explain why the suppression 
the `econnrefused` error is no included in spec